src/calendar-heatmap.js

```diff
@@ -1,20 +1,26 @@
 
 function calendarHeatmap() {
   // defaults
+  var SQUARE_LENGTH = 10;
   var width = 750;
+  width = SQUARE_LENGTH * (52+12);
```

Comment:
I have tried to determine the svg size by square width. It can be done opposite, i mean, determine square width by svg size..